src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.margin-form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
move to `html` selector

Reply:
If I'm moving my font-family to html selector, all fonts are stop working in the code, what is the reason? I just copypaste from class selector to html, but nothing is happening.